pkg/build/buildkit/connector/portforward.go

```diff
@@ -48,9 +49,11 @@ type PortForwarderOktetoContextInterface interface {
 
 // forwarder represents a port forwarder for the buildkit server.
 type forwarder struct {
```

Comment:
The fact that this struct and the "public" one have the same description is a smell of something odd, or not right. Why do we need a private struct that has some of the functionality that the public one would have?